map_server/src/main.cpp

```diff
@@ -43,6 +43,7 @@
 #include <memory>
 
 #include <rclcpp/rclcpp.hpp>
+#include <rcutils/logging_macros.h>
```

Comment:
I think you need to update the `package.xml` and `CMakeLists.txt` to depend on and use `rcutils` since it's being used directly.